lib/pages/home/pages/search/search.dart

```diff
@@ -70,26 +70,26 @@ class OBMainSearchPageState extends State<OBMainSearchPage> {
     _navigationService = openbookProvider.navigationService;
     _localizationService = openbookProvider.localizationService;
 
-    Widget currentWidget;
-
-    if (_hasSearch) {
-      currentWidget = OBUserSearchResults(
-        searchQuery: _searchQuery,
-        userResults: _userSearchResults,
-        userSearchInProgress: _userSearchRequestInProgress,
-        communityResults: _communitySearchResults,
-        communitySearchInProgress: _communitySearchRequestInProgress,
-        onUserPressed: _onSearchUserPressed,
-        onCommunityPressed: _onSearchCommunityPressed,
-        selectedTab: _selectedSearchResultsTab,
-        onScroll: _onScroll,
-        onTabSelectionChanged: _onSearchTabSelectionChanged,
-      );
-    } else {
-      currentWidget = OBTrendingPosts(
-        controller: _trendingPostsController,
-      );
-    }
+    Widget indexedStackWidget = IndexedStack(
```

Comment:
Cool widget!

Reply:
Yea I see a bright future for this widget 👯‍♂